shared/oae/api/oae.api.util.js

```diff
@@ -701,6 +701,11 @@ define(['exports', 'require', 'jquery', 'underscore', 'oae.api.config', 'markdow
             if (/https?:\/\//i.test(displayName)) {
                 return false;
             }
+            
+            // Display names that contain `@` are also considered invalid
```

Comment:
Display names containing an `@` sign are also indicative of Shibboleth not releasing an attribute that could be used as the display name. Can we change this to:

```
// Display names that contain `http://`, `https://` or `@` are indicative of Shibboleth
// not releasing an attribute that could be used as the display name. This is not
// considered to be valid
if (/https?:\/\//i.test(displayName)) {
    return false;
} else if (/@/.test(displayName)) {
    return false;
}
```